packages/qwik/src/core/render/ssr/render-ssr.ts

```diff
@@ -655,7 +655,7 @@ This goes against the HTML spec: https://html.spec.whatwg.org/multipage/dom.html
       }
     }
     if (key != null) {
-      openingElement += ' q:key="' + key + '"';
+      openingElement += ' q:key="' + key.replace(/"/g, '&quot;') + '"';
```

Comment:
just have seen that there is already a fn which can be used for the escaping:

```suggestion
openingElement += ' q:key="' + escapeAttr(key) + '"';
```

Reply:
@zanettin is right here! we should use the escapeAttr() function